src/components/Swap/Swap.tsx

```diff
@@ -31,19 +31,19 @@ export interface SwapToken {
 export interface Pools {
```

Comment:
there shouldn't be any changes that are not included in scope of task, unless they're necessary for something

Reply:
My code formatting extension formatted the file and I revert it already